src/config/mailConfig.js

```diff
@@ -0,0 +1,42 @@
+import nodemailer from 'nodemailer';
+import dotenv from 'dotenv';
+
+dotenv.config();
+
+const email = process.env.EMAIL;
+const host = process.env.SMTP_HOST;
+const port = process.env.SMTP_PORT;
+const username = process.env.SMTP_USERNAME;
+const pass = process.env.SMTP_MAIL_PASSWORD;
+
+const transporter = nodemailer.createTransport({
+  host,
+  port,
+  auth: {
+    user: username,
+    pass
+  }
+});
+
+const sendMail = (req, res, body) => {
+  const {
+    receivers, subject, text, html
+  } = body;
+  const allRecipientEmail = receivers.join(', ');
+  const mailOptions = {
+    from: email,
+    to: allRecipientEmail,
+    subject,
+    text,
+    html,
+  };
+
+  transporter.sendMail(mailOptions, (error, info) => {
```

Comment:
let's use `async/await` here instead of callbacks.

Reply:
noted

Reply:
can you also write tests for this while you at that?

Reply:
okay